README.md

```diff
@@ -1,149 +1,43 @@
-# Getting started example for Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# Blinky Mbed OS Example
 
-This guide reviews the steps required to get Blinky with the addition of dynamic OS statistics working on an Mbed OS platform. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).)
+The example project is part of the [Arm Mbed OS quick start guide](https://os.mbed.com/docs/mbed-os/v5.14/quick-start/index.html). It contains an application to repeatedly blink an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
 
-Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+The project can be built with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli). Please install Arm Mbed CLI.
 
-## Import the example application
+Depending on the target, the example project can be built with `GCC_ARM`, `ARM` or `IAR` toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain supports your target.
 
-From the command-line, import the example:
-
-```
-mbed import mbed-os-example-blinky
-cd mbed-os-example-blinky
-```
-
-### Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the Arm Compiler:
-
-```
-mbed compile -m K64F -t ARM
+```bash
+$ mbed compile -S
 ```
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
-
-```
-[snip]
-
-Image: ./BUILD/K64F/GCC_ARM/mbed-os-example-blinky.bin
-```
-
-### Program your board
-
-1. Connect your Mbed device to the computer over USB.
-1. Copy the binary file to the Mbed device.
-1. Press the reset button to start the program.
+Clone this repository on your system and change the current directory to where the project was cloned.
```

Comment:
Why not keep the original instruction to import the example application with command-line snippet?
I would also keep the "Import the example application" section.

Reply:
Read here:
https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/#discussion_r334481963

Reply:
I can't see a specific comment from the link above

Reply:
Here: https://github.com/ARMmbed/mbed-os-example-blinky/pull/192#discussion_r334481963

Reply:
[Added commands to download the project using `git clone` and `mbed import`](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/1b45642257997c208517f49089b6b306f2661fcd)